examples/app-cpp/README.md

```diff
@@ -3,8 +3,18 @@
 This directory contains an example Payload Application using the C++.
 This is not production-ready code, but it is useful for learning how the platform functions.
 
+NOTE: All the cpp files need to have .cc extention only.
+
```

Comment:
Add here as:

# Build cpp application
To build application in examples/app-cpp , go to SatOS_Payload_SDK parent directory and run "make build_cpp"

`make build_cpp` will instantiate a build environment container and compile c++ application present at examples/app-cpp.

```
cd ../.. 
make build_cpp
```

Reply:
Done